src/operator/timeoutWith.ts

```diff
@@ -43,12 +42,13 @@ class TimeoutWithSubscriber<T, R> extends OuterSubscriber<T, R> {
     return this._hasCompleted;
   }
 
-  constructor(destination: Subscriber<T>,
+  constructor(public destination: Subscriber<T>,
               private absoluteTimeout: boolean,
               private waitFor: number,
               private withObservable: Observable<any>,
               private scheduler: Scheduler) {
-    super(destination);
+    super(null);
+    destination.add(this);
```

Comment:
Yesterday I pulled it down and ran it and it looked fine. I just didn't see that bit.